zkevm-circuits/src/sig_circuit.rs

```diff
@@ -533,26 +521,35 @@ impl<F: Field> SignVerifyChip<F> {
         // ================================================
         // message hash cells
         // ================================================
-        let assigned_msg_hash_le = (!padding)
-            .then(|| sign_data.msg_hash.to_bytes())
-            .unwrap_or_default()
-            .iter()
-            .map(|&x| QuantumCell::Witness(Value::known(F::from_u128(x as u128))))
-            .collect_vec();
+
+        let assert_crt = |ctx: &mut Context<F>,
+                          bytes: [u8; 32],
+                          crt_integer: &CRTInteger<'v, F>,
+                          overriding: &Option<&QuantumCell<F>>|
+         -> Result<_, Error> {
+            //let bytes: [u8; 32] = v.to_bytes();
+            let byte_cells: Vec<QuantumCell<F>> = bytes
+                .iter()
+                .map(|&x| QuantumCell::Witness(Value::known(F::from_u128(x as u128))))
```

Comment:
change to `F::from(x as u64)` 
this allows for a bit faster allocation with witness-gen optimization